modules/core/src/v2/coins/eos.ts

```diff
@@ -408,4 +438,268 @@ export class Eos extends BaseCoin {
       .call(this)
       .asCallback(callback);
   }
+
+  /**
+   * Validate a public or private key
+   * If passphrase is provided, try to decrypt the key with it
+   * @param key
+   * @param source
+   * @param passphrase
+   * @param isUnsignedSweep
+   * @param isKrsRecovery
+   */
+  validateKey = function({ key, source, passphrase, isUnsignedSweep, isKrsRecovery }: ValidateKeyParams): HDNode {
```

Comment:
types for params

Reply:
Types are in `ValidateKeyParams`. However, the pattern we have  been following is to use the suffix `Options`,  so this should be `ValidateKeyOptions`.